public/js/womenFashion.js

```diff
@@ -0,0 +1,5 @@
+function btnClick() {
```

Comment:
we have 2 function's do the same in menFashion .. so put one of them inside file and use it! why create the same function twice !